monitoring/system-test/quickstart.test.js

```diff
@@ -23,7 +23,8 @@ const client = proxyquire(`@google-cloud/monitoring`, {}).v3().metricServiceClie
 test.beforeEach(stubConsole);
 test.afterEach.always(restoreConsole);
 
-test.cb(`should list time series`, (t) => {
+// TODO(anassri): Fix this test occasionally omitting time series data
```

Comment:
Don't put TODOs in code, you have *two* world-class bug tracking tools at your disposal.